sklearn/utils/weight_vector.pxd

```diff
@@ -8,8 +8,6 @@ cdef extern from "math.h":
 
 
 cdef class WeightVector(object):
-    cdef np.ndarray w
-    cdef np.ndarray aw
```

Comment:
Technically the class is public, but I guess it shouldn't have been?

Reply:
yes please :) https://github.com/scikit-learn/scikit-learn/issues/6616#issuecomment-516479413